rlberry/envs/basewrapper.py

```diff
@@ -33,6 +34,7 @@ def __init__(self, env, wrap_spaces=False):
         # Save reference to env
         self.env = env
         self.metadata = self.env.metadata
+        self.render_mode = self.env.render_mode
```

Comment:
Maybe we can add this in an attribute section of docstring ? I think it would be nice to progressively add it everywhere.